src/commands/conferences/create.ts

```diff
@@ -28,7 +28,7 @@ export class conferencesCreate extends Command {
         waitUrl: flags.string({
             char: "w",
             description:
-                "If specified, a URL for the audio file that provides custom hold music for the Conference when it is in the populated state. Otherwise, FreeClimb uses a system default audio file. This is always fetched using HTTP GET and is fetched just once — when the Conference is created.",
+                "If specified, a URL for the audio file that provides custom hold music for the Conference when it is in the populated state. Otherwise, FreeClimb uses a system default audio file. This is always fetched using HTTP GET and is fetched just once &mdash; when the Conference is created.",
```

Comment:
Override to avoid HTML entity

Reply:
common html entiities are now removed in generation of the schema